README.md

```diff
@@ -140,6 +140,47 @@ transactionRequest.run(new ResultCallback<TransactionId>() {
 });
 ```
 
+#####Memo
+Arbitrary data can be added to a transfer operation using the memo parameter,
+the memo is a `String` of up to 28 characters.
+
+```java
+String memo = "arbitrary data";
+transactionRequest = account.sendTransaction(toAddress, getPassphrase(), amount, memo);
+transactionRequest.run(new ResultCallback<TransactionId>() {
+
+    @Override
+        public void onResult(TransactionId result) {
+            Log.d("example", "The transaction id: " + result.toString());
+        }
+
+        @Override
+        public void onError(Exception e) {
+            e.printStackTrace();
+        }
+});
+```
+### Watching Payments
+
+Ongoing payments in KIN, from or to an account, can be observed. 
+<br/>First, Create a `PaymentWatcher`
+object:
+```java
+PaymentWatcher watcher = account.createPaymentWatcher();
+```
+For start watching, use `start` method, by providing listener of `WatcherListener<PaymentInfo>`:
+```java
+paymentWatcher.start(new WatcherListener<PaymentInfo>() {
```

Comment:
do you need to specify that calling start can't be called again unless stop is called

Reply:
yes, it's not supported currently I'll change it